code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -278,8 +277,8 @@
 	else
 		M.apply_damage(0.6*reac_volume, BRUTE)
 
-/datum/reagent/blob/explosive_lattice/damage_reaction(obj/structure/blob/B, original_health, damage, damage_type, cause)
-	if(isnull(cause))
+/datum/reagent/blob/explosive_lattice/damage_reaction(obj/structure/blob/B, damage, damage_type, damage_flag)
+	if(damage_flag != "melee" || damage_flag != "bullet" || damage_flag != "laser")
```

Comment:
These lines could be simplified/made proper, now that there's an actual way to tell if a bomb is hitting it.

    if(damage_flag == "bomb")
    	return 0
    else if(damage_flag != "melee" || damage_flag != "bullet" || damage_flag != "laser")
    	return damage * 1.5
    return ..()